Magento_Theme/templates/header/search.phtml

```diff
@@ -0,0 +1,46 @@
+<?php
+// @codingStandardsIgnoreFile
+
+/** @var $block \Magento\Framework\View\Element\Template */
+/** @var $helper \Magento\Search\Helper\Data */
+$helper = $this->helper('Magento\Search\Helper\Data');
+?>
+
+<div class="header__search-wrapper">
+    <form class="search-form"
+          id="search_mini_form"
+          action="<?= /* @escapeNotVerified */ $helper->getResultUrl() ?>"
+          method="get"
+    >
+        <div class="input search-form__input-wrapper">
+            <label class="label search-form__label" for="search">
+                <?= __('Search') ?>
+            </label>
+            <input id="search"
+                   data-mage-init='{"quickSearch":{
+                        "formSelector":"#search_mini_form",
+                        "url":"<?= /* @escapeNotVerified */ $block->getUrl('search/ajax/suggest', ['_secure' => $block->getRequest()->isSecure()]); ?>",
+                        "destinationSelector":"#search_autocomplete"}
+                   }'
+                   type="text"
+                   name="<?= /* @escapeNotVerified */ $helper->getQueryParamName(); ?>"
+                   value="<?= /* @escapeNotVerified */ $helper->getEscapedQueryText(); ?>"
+                   placeholder="<?= /* @escapeNotVerified */ __('Search entire store here...'); ?>"
+                   class="input__field search-form__input"
+                   maxlength="<?= /* @escapeNotVerified */ $helper->getMaxQueryLength(); ?>"
+                   role="combobox"
+                   aria-haspopup="false"
+                   aria-autocomplete="both"
```

Comment:
if we don't have autocomplete there should be: `aria-autocomplete="none"`

Reply:
Changed